spec/preprocessorSpec.js

```diff
@@ -0,0 +1,33 @@
+var reqTest = require('../js/analyzer.js');
```

Comment:
Do you need to store requires in a variable? The variable itself doesn't seem to be used.